index.html

```diff
@@ -1,59 +1,67 @@
 <!DOCTYPE html>
-<html>
+<html lang="fr">
 
 <head>
     <meta charset="utf-8" />
-    <link rel="stylesheet" href="style.scss" />
-    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="public/css/style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Yesteryear&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Zhi+Mang+Xing&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Inter&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/990c8adfeb.js" crossorigin="anonymous"></script>
     <title>ohmyfood</title>
 </head>
 
 <body>
     <header>
-        <div class="menu">
-            <h1>ohmyfood</h1>
-        </div>
+        <h1>ohmyfood</h1>
     </header>
 
-    <section>
-        <div class="menus">
-            <button class="menu">
-                <h2>La note enchantée</h2>
-                Menu
-            </button>
-
-            <button class="menu">
-                <h2>Le chic à la française</h2>
-                Menu
-            </button>
-        </div>
-
-        <div class="menus">
-            <button class="menu">
-                <h2>La pelette du goût</h2>
-                Menu
-            </button>
-
-            <button class="menu">
-                <h2>La palette du goût</h2>
-                Menu
-            </button>
-        </div>
-
-
-        <button class="prochainement">
+    <nav class="menus">
+        <ul>
+            <li>
+                <a href="page1.html"><i class="fas fa-music"></i>
```

Comment:
Tu connais `aria-hidden`?

https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility